src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/disk_encryption.py

```diff
@@ -39,6 +40,15 @@
     }
 }
 
+def _get_extension_info(os_type):
```

Comment:
I suggest we remove this method, rather just apply on the dictionary above such as :

```python
vm_extension_info = {
     'Linux': {
         'publisher': os.environ.get('ADE_TEST_EXTENSION_PUBLISHER') or 'Microsoft.Azure.Security',
         'name': os.environ.get('ADE_TEST_EXTENSION_NAME') or 'AzureDiskEncryptionForLinux',
         'version': '0.1'
     },
     'Windows': {
         'publisher': os.environ.get('ADE_TEST_EXTENSION_PUBLISHER') or 'Microsoft.Azure.Security',
         'name': os.environ.get('ADE_TEST_EXTENSION_NAME') or 'AzureDiskEncryption',
         'version': '1.1'
     }
 }
 ```